src/layout.js

```diff
@@ -28,7 +28,6 @@ var Layout = function(optParams) {
   Section.call(this, params);
 
   this.type = params.type;
-  this.dom.setAttribute('contenteditable', false);
```

Comment:
This is also a breaking change. This allows the cursor to be inside layout container. You can see this is breaking by staging an image and then sitting the cursor at a paragraph after that stage and click the back arrow to see the cursor moving into that layout and user would be able to type outside the editing area.